components/src/molecules/message/Message.scss

```diff
@@ -0,0 +1,94 @@
+@import '../../../assets/styles/globals';
+
+$message__max-width\@medium    : 320px !default;
+$message__width                : 100% !default;
+$message__content-width        : 90% !default;
+$message__padding              : $spacer--medium 24px 0 24px !default;
+$message__information-padding  : 0 0 $spacer--medium 0 !default;
+$message__font-size            : $font-size-small !default;
+$message__transition           : $transition-base !default;
+$message__color                : $white !default;
+$message__color-success        : $black !default;
+$message__background           : #1976d2 !default;
+$message__background--success  : #69dca1 !default;
+$message__background--error    : #d32f2f !default;
+$message__border-success       : 1px solid rgba(57, 50, 67, 0.4) !default;
```

Comment:
the same here, should be `$message__border-bottom--success `, right? and move below `$message__border-bottom`